wgengine/router/router_linux.go

```diff
@@ -891,6 +894,46 @@ func (r *linuxRouter) downInterface() error {
 	return netlink.LinkSetDown(link)
 }
 
+// fixupWSLMTU fixes the MTU on the WSL eth0 interface to be at least 1360 bytes
+// iff the running system appears to be in a common/well-known WSL
```

Comment:
I know what "iff" means but from 10 years of Go reviews, empirically nobody else does and we get "typo" fixes forever about them. So just spell it out. :(